src/Routes.js

```diff
@@ -35,10 +35,10 @@ class Routes extends Component {
           subId={subCategoryId}
         />
         <Switch>
-          <Route exact path="/main" component={Main} />
+          <Route exact path="/" component={Main} />
           <Route
             exact
-            path="/list/category=:categoryId/subcateogry=:subcategoryId"
+            path="/list/category=:categoryId/subcategory=:subcategoryId"
```

Comment:
path형식을 이렇게 표현하신 이유가 있나요? category=:categoryID 이런식으로 표현하는건 query의 형식입니다.
path로 설정하기에 알맞지 않습니다.
```suggestion
            path="/list/:categoryId/:subcategoryId"
```
이렇게 해도 되지 않을까요?

Reply:
앗 path와 query의 개념을 햇갈렸네요. 모두 수정 완료했습니다!!